generate.go

```diff
@@ -6,3 +6,6 @@ package authd
 //
 // Should it become default, remove the --experimental_allow_proto3_optional flag from the go generate command below.
 //go:generate sh -c "PATH=\"$PATH:`go env GOPATH`/bin\" protoc --proto_path=. --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative authd.proto --experimental_allow_proto3_optional"
+
+// Builds the NSS crate in order to regenerate the GRPC stubs for Rust.
+//go:generate sh -c "cargo build"
```

Comment:
I'm still not sure if we need it, tbh. The compiled files go under the `target/` directory (and we definitely do not want this dir being pushed to the repo), so we'll need some more investigation on it. I know that the output directory (for the compiled proto) can be overridden, but this relies on an environment variable and then the linking gets really weird (since the macro `tonic::include_proto!` only works if the proto file is in the `target/` directory)

Reply:
ok, let’s not include it for now